Compiling error

Good morning. Good to read you rewrote the itho fancontrol. i was trying to find a solution and here it is.
only thin, i get a compiling error.

''
Compiling /data/fancontrol/.pioenvs/fancontrol/libd10/esphome_itho/CC1101.cpp.o
In file included from /data/fancontrol/.piolibdeps/fancontrol/esphome_itho/CC1101.cpp:5:
/data/fancontrol/.piolibdeps/fancontrol/esphome_itho/CC1101.h:10:10: fatal error: SPI.h: No such file or directory

*************************************************************
* Looking for SPI.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:SPI.h"
* Web  > https://registry.platformio.org/search?q=header:SPI.h
*
*************************************************************

   10 | #include <SPI.h>
      |          ^~~~~~~
compilation terminated.
*** [/data/fancontrol/.pioenvs/fancontrol/libd10/esphome_itho/CC1101.cpp.o] Error 1
========================= [FAILED] Took 325.43 seconds =========================
''

--- a/config/esphome/itho.yaml
+++ b/config/esphome/itho.yaml
@@ -5,9 +5,10 @@ esphome:
   includes: 
     - ITHO/fancontrol.h
   libraries: 
+    - SPI  
     - https://github.com/CoMPaTech/esphome_itho.git
     - Ticker
-    - SPI
+
 
 wifi:
   ssid: "secret"

